src/modules/handlers/screen/poster/poster.ts

```diff
@@ -29,7 +29,8 @@ export interface BasePoster {
   timeout: number;
   footer: FooterSize;
   /** Whether this poster should only be shown when in BorrelMode */
-  borrelMode: boolean;
+  borrelMode?: boolean;
+  color?: string;
```

Comment:
In a perfect world, we separate this as two separate types. However, that would also mean that the `TrelloPosterManager` should support the parsing of different type of components (for example with callbacks). Sounds like a lot of work, so let's keep that out of scope for now (unless GEWIS and Hubble posters start to deviate a lot).

Reply:
I did that initially as well cause I felt the same, but that created a super complex typing system, that also propagates to the frontend. But indeed something to keep in mind for the future.